index.html

```diff
@@ -317,7 +317,7 @@
                         </dd>
                     <dt><dfn>pressure</dfn></dt>
                         <dd>
-                            <p>The normalized pressure of the pointer input in the range of [0,1], where 0 and 1 represent the minimum and maximum pressure the hardware is capable of detecting, respectively. For hardware that does not support pressure, the value MUST be 0.5 when in the <a>active buttons state</a> and 0 otherwise.</p>
+                            <p>The normalized pressure of the pointer input in the range of [0,1], where 0 and 1 represent the minimum and maximum pressure the hardware is capable of detecting, respectively. For hardware that does not support pressure, the value MUST be 0.5 when in the <a>active buttons state</a> and 0 otherwise. Particularly as a result, all <code>pointerup</code> events will have pressure 0.</p>
```

Comment:
I think we can get away with dropping the "Particularly as a result" (which reads a bit funny) and instead change this to, say, "Note: all <code>pointerup</code> ..."